datafusion/src/physical_plan/sort_preserving_merge.rs

```diff
@@ -1183,7 +1191,7 @@ mod tests {
     async fn test_async() {
         let schema = test::aggr_test_schema();
         let sort = vec![PhysicalSortExpr {
-            expr: col("c7", &schema).unwrap(),
+            expr: col("c12", &schema).unwrap(),
```

Comment:
this is needed because `c7` is not unique and this since `arrow-rs` 6.0 sort is no longer stable the output is not deterministic.